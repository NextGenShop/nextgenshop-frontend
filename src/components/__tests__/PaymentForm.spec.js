import React from 'react';
import { shallow } from 'enzyme';
import PaymentForm from '../PaymentForm';

describe('PaymentForm', () => {
  test('should correctly render', () => {
    const component = shallow(<PaymentForm />);
    expect(component).toMatchSnapshot();
  });
});
