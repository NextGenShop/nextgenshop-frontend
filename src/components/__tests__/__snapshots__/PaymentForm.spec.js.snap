// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentForm should correctly render 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Typography))
    gutterBottom={true}
    variant="h6"
  >
    Payment method
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Grid))
    container={true}
    spacing={3}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
      xs={12}
    >
      <WithStyles(ForwardRef(TextField))
        autoComplete="cc-name"
        fullWidth={true}
        id="cardName"
        label="Name on card"
        required={true}
      />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
      xs={12}
    >
      <WithStyles(ForwardRef(TextField))
        autoComplete="cc-number"
        fullWidth={true}
        id="cardNumber"
        label="Card number"
        required={true}
      />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
      xs={12}
    >
      <WithStyles(ForwardRef(TextField))
        autoComplete="cc-exp"
        fullWidth={true}
        id="expDate"
        label="Expiry date"
        required={true}
      />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
      xs={12}
    >
      <WithStyles(ForwardRef(TextField))
        autoComplete="cc-csc"
        fullWidth={true}
        helperText="Last three digits on signature strip"
        id="cvv"
        label="CVV"
        required={true}
      />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={12}
    >
      <WithStyles(ForwardRef(FormControlLabel))
        control={
          <WithStyles(ForwardRef(Checkbox))
            color="secondary"
            name="saveCard"
            value="yes"
          />
        }
        label="Remember credit card details for next time"
      />
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</Fragment>
`;
